notebooks/MAST/plot_analyze_with_lightkurve/plot_analyze_with_lightkurve.ipynb

```diff
@@ -0,0 +1,953 @@
+{
```

Comment:
<p>Yes, I think you definitely want to filter out the data points that don't have quality == 0.</p><p> </p><p>Could you just do something like this?:</p><pre class="ql-syntax" spellcheck="false">lc_stitched = lc_stitched[lc_stitched['quality']==0]
lc_stitched.flux=lc_stitched['kspsap_flux']
lc_stitched.flux_err=lc_stitched['kspsap_flux_err']
lc_stitched.meta['FLUX_ORIGIN']='kspsap_flux'
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
<p>Such an easy fix and it worked :-) </p>